<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Библиотека</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="library_page">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTabWidget" name="library_tabWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="reader_tab">
           <attribute name="title">
            <string>Читатели</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QTableWidget" name="reader_tableWidget">
                <property name="rowCount">
                 <number>0</number>
                </property>
                <property name="columnCount">
                 <number>0</number>
                </property>
                <attribute name="horizontalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderCascadingSectionResizes">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>150</number>
                </attribute>
                <attribute name="horizontalHeaderHighlightSections">
                 <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderDefaultSectionSize">
                 <number>25</number>
                </attribute>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <property name="topMargin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="search_label">
                    <property name="text">
                     <string>Введите для поиска:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="search_lineEdit"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="search_reader_pushButton">
                    <property name="text">
                     <string>Найти</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QGroupBox" name="search_parameters_groupBox">
                  <property name="title">
                   <string>Поиск по:</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout">
                   <item row="0" column="0" colspan="2">
                    <widget class="QRadioButton" name="search_by_FIO_radioButton">
                     <property name="text">
                      <string>ФИО</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0" colspan="2">
                    <widget class="QRadioButton" name="search_by_card_number_radioButton">
                     <property name="text">
                      <string>№ читательского билета</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="add_reader_pushButton">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Добавить...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="delete_reader_pushButton">
                <property name="text">
                 <string>Удалить</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="delete_all_readers_pushButton">
                <property name="text">
                 <string>Стерерть всех</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="show_all_readers_pushButton">
                <property name="text">
                 <string>Показать всех</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
           <zorder>reader_tableWidget</zorder>
           <zorder></zorder>
           <zorder>search_parameters_groupBox</zorder>
           <zorder>search_reader_pushButton</zorder>
           <zorder>search_lineEdit</zorder>
           <zorder>search_label</zorder>
           <zorder>delete_all_readers_pushButton</zorder>
          </widget>
          <widget class="QWidget" name="book_tab">
           <attribute name="title">
            <string>Книги</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTableWidget" name="book_tableWidget">
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>150</number>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>25</number>
              </attribute>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="add_book_pushButton">
                <property name="text">
                 <string>Добавить...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="delete_book_pushButton">
                <property name="text">
                 <string>Удалить</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="delete_all_books_pushButton">
                <property name="text">
                 <string>Стерерть все</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="search_book_pushButton">
                <property name="text">
                 <string>Поиск...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="iobooksinfo_tab">
           <attribute name="title">
            <string>Прием/Выдача</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QTableWidget" name="iobooksinfo_tableWidget">
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>150</number>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>25</number>
              </attribute>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="take_iobooksinfo_pushButton">
                <property name="text">
                 <string>Принять книгу</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="give_iobooksinfo_pushButton">
                <property name="text">
                 <string>Выдать книгу</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="create_reader_page">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="cr_right_of_acces_label">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>Права доступа:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cr_right_of_acces_comboBox">
          <property name="maxVisibleItems">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>только абонемент (А)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>только читальный зал (Ч)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>читальный зал и абонемент (В)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="cr_fio_label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ФИО:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="cr_fio_lineEdit">
          <property name="inputMask">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="cr_year_of_birth_label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Год рождения:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="cr_adress_label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Адрес:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="cr_job_or_study_place_label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Место работы или учебы:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="cr_job_or_study_place_lineEdit"/>
        </item>
        <item row="5" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLineEdit" name="cr_year_of_birth_lineEdit">
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="maxLength">
               <number>32767</number>
              </property>
              <property name="cursorPosition">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="10" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cr_ok_pushButton">
            <property name="text">
             <string>Ok</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cr_cancel_pushButton">
            <property name="text">
             <string>Отмена</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="cr_adress_lineEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="library_menu">
    <property name="title">
     <string>Библиотека</string>
    </property>
    <addaction name="separator"/>
    <addaction name="create_library_action"/>
    <addaction name="open_library_action"/>
    <addaction name="separator"/>
    <addaction name="save_library_action"/>
    <addaction name="save_as_library_action"/>
    <addaction name="separator"/>
    <addaction name="quit_library_action"/>
   </widget>
   <widget class="QMenu" name="reader_menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Читатель</string>
    </property>
    <addaction name="add_reader_action"/>
    <addaction name="delete_reader_action"/>
    <addaction name="separator"/>
    <addaction name="delete_all_readers_action"/>
    <addaction name="separator"/>
    <addaction name="show_all_readers_action"/>
   </widget>
   <widget class="QMenu" name="book_menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Книга</string>
    </property>
    <addaction name="add_book_action"/>
    <addaction name="delete_book_action"/>
    <addaction name="separator"/>
    <addaction name="delete_all_books_action"/>
    <addaction name="separator"/>
    <addaction name="search_book_action"/>
   </widget>
   <widget class="QMenu" name="iobooksinfo_menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Прием\Выдача</string>
    </property>
    <addaction name="take_iobooksinfo_action"/>
    <addaction name="give_iobooksinfo_action"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="about_help_action"/>
   </widget>
   <addaction name="library_menu"/>
   <addaction name="reader_menu"/>
   <addaction name="book_menu"/>
   <addaction name="iobooksinfo_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="create_library_action">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="save_library_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="add_reader_action">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Добавить...</string>
   </property>
  </action>
  <action name="delete_reader_action">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="delete_all_readers_action">
   <property name="text">
    <string>Стереть всех</string>
   </property>
  </action>
  <action name="show_all_readers_action">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Показать всех</string>
   </property>
  </action>
  <action name="add_book_action">
   <property name="text">
    <string>Добавить...</string>
   </property>
  </action>
  <action name="delete_book_action">
   <property name="text">
    <string>Удалить...</string>
   </property>
  </action>
  <action name="delete_all_books_action">
   <property name="text">
    <string>Стереть все</string>
   </property>
  </action>
  <action name="search_book_action">
   <property name="text">
    <string>Поиск...</string>
   </property>
  </action>
  <action name="take_iobooksinfo_action">
   <property name="text">
    <string>Принять книгу</string>
   </property>
  </action>
  <action name="give_iobooksinfo_action">
   <property name="text">
    <string>Выдать книгу</string>
   </property>
  </action>
  <action name="save_as_library_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="open_library_action">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="quit_library_action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="about_help_action">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>reader_tableWidget</tabstop>
  <tabstop>add_reader_pushButton</tabstop>
  <tabstop>delete_reader_pushButton</tabstop>
  <tabstop>delete_all_readers_pushButton</tabstop>
  <tabstop>show_all_readers_pushButton</tabstop>
  <tabstop>book_tableWidget</tabstop>
  <tabstop>add_book_pushButton</tabstop>
  <tabstop>delete_book_pushButton</tabstop>
  <tabstop>delete_all_books_pushButton</tabstop>
  <tabstop>search_book_pushButton</tabstop>
  <tabstop>iobooksinfo_tableWidget</tabstop>
  <tabstop>take_iobooksinfo_pushButton</tabstop>
  <tabstop>give_iobooksinfo_pushButton</tabstop>
  <tabstop>cr_right_of_acces_comboBox</tabstop>
  <tabstop>cr_fio_lineEdit</tabstop>
  <tabstop>cr_year_of_birth_lineEdit</tabstop>
  <tabstop>cr_adress_lineEdit</tabstop>
  <tabstop>cr_job_or_study_place_lineEdit</tabstop>
  <tabstop>cr_ok_pushButton</tabstop>
  <tabstop>cr_cancel_pushButton</tabstop>
  <tabstop>library_tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
